azure-pipelines.yaml

```diff
@@ -54,15 +54,14 @@ steps:
   # extracts merged PR number from git log if a PR merge to master
 - script: |
     gitLog=`git log -1 --pretty=%B`
-    mergedPrNo=`echo $gitLog | sed -n 's/Merge pull request #\(.*\) from .*/\1/p'`
+    mergedPrNo=`echo $gitLog | sed -n 's/.*(#\([0-9]\+\)).*/\1/p'`
     echo '##vso[task.setvariable variable=mergedPrNo]'$mergedPrNo
```

Comment:
The logic can be tested by on the master branch of this repository. The following should print out 11, the last branch merged
```
gitLog=`git log -1 --pretty=%B`
mergedPrNo=`echo $gitLog | sed -n 's/.*(#\([0-9]\+\)).*/\1/p'` 
echo $mergedPrNo
```